packages/docusaurus-theme-classic/src/theme/CodeBlock/Content/String.tsx

```diff
@@ -74,16 +75,17 @@ export default function CodeBlockString({
       {title && <div className={styles.codeBlockTitle}>{title}</div>}
       <div className={styles.codeBlockContent}>
         <Highlight
-          {...defaultProps}
+          prism={Prism}
```

Comment:
I think this is the issue, we used the "vendored prism library" and should keep using it so passing a prism instance is not needed now. I'll remove it.